test/test_network_constructors.jl

```diff
@@ -178,6 +178,123 @@ end
     end
 end
 
+@testset "Network DC-PF with VirtualPTDF Model and implementing Dynamic Branch Ratings" begin
+    line_device_model = DeviceModel(
+        Line,
+        StaticBranch;
+        time_series_names = Dict(
+            DynamicBranchRatingTimeSeriesParameter => "dynamic_line_ratings",
+        ))
+    TapTransf_device_model = DeviceModel(
+        TapTransformer,
+        StaticBranch;
+        time_series_names = Dict(
+            DynamicBranchRatingTimeSeriesParameter => "dynamic_line_ratings",
+        ))
+    c_sys5 = PSB.build_system(PSITestSystems, "c_sys5")
+    c_sys14 = PSB.build_system(PSITestSystems, "c_sys14")
+    c_sys14_dc = PSB.build_system(PSITestSystems, "c_sys14_dc")
+    systems = [c_sys5, c_sys14, c_sys14_dc]
+    objfuncs = [GAEVF, GQEVF, GQEVF]
+    constraint_keys = [
+        PSI.ConstraintKey(RateLimitConstraint, PSY.Line, "lb"),
+        PSI.ConstraintKey(RateLimitConstraint, PSY.Line, "ub"),
+        PSI.ConstraintKey(CopperPlateBalanceConstraint, PSY.System),
+        PSI.ConstraintKey(NetworkFlowConstraint, PSY.Line),
+    ]
+    PTDF_ref = IdDict{System, VirtualPTDF}(
+        c_sys5 => VirtualPTDF(c_sys5),
+        c_sys14 => VirtualPTDF(c_sys14),
+        c_sys14_dc => VirtualPTDF(c_sys14_dc),
+    )
+    branches_dlr = IdDict{System, Vector{String}}(
+        c_sys5 => ["1", "2", "6"],
+        c_sys14 => ["Line1", "Line2", "Line9", "Line10", "Line12", "Trans2"],
+        c_sys14_dc => ["Line1", "Line9", "Line10", "Line12", "Trans2"],
+    )
+    dlr_factors = vcat([fill(x, 6) for x in [1.15, 1.05, 1.1, 1]]...)
+    test_results = IdDict{System, Vector{Int}}(
+        c_sys5 => [264, 0, 264, 264, 168],
```

Comment:
Where did all these numbers come from? If they are ground truth computed by hand, write some comments about how they were computed. If they are the outputs of the model the first time it was run, we need more ground truthy tests.